src/Output.js

```diff
@@ -7,11 +7,14 @@ module.exports = class extends ChainedMap {
       'auxiliaryComment',
       'chunkFilename',
       'chunkLoadTimeout',
+      'chunkCallbackName',
```

Comment:
This is super nitpicky, but could you move this to be before `chunkFilename` so it stays in alphabetical order?

Reply:
There's a bit of a mix throughout webpack-chain at the moment of alphabetical order vs schema order - would be good to sort it all out at some point :-)

Reply:
Oh, I thought I already cleaned all those up... 🤔

Reply:
Ok, I did a very quick run through to get all of the `extend()` calls at least to specify in alphabetical order.

+1 to setting up linting and/or Prettier 😄